README.md

```diff
@@ -86,9 +86,37 @@ Add a new tab to the tab group and returns a `Tab` instance.
 * `active` (default: `false`): set this to `true` if you want to activate the tab once it is loaded. Otherwise you will need to call `tab.activate()`.
 * `ready`: a callback function to call once the tab is ready. The `Tab` instance is passed as the only parameter.
 
-#### `tabGroup.getTab(id)`
+#### `tabGroup.getTab(id, idIsPosition)`
```

Comment:
I think this alias introduces unnecessary complexity. I would recommend to keep only two simple methods:

* `tabGroup.getTab(id)` which gets the tab from its id.
* `tabGroup.getTabByPosition(position)` which gets the tab from its position.

But maybe missed is a use case for `tabGroup.getTab(id, idIsPosition)`?

Reply:
Okay, that makes sense. I will do that.